src/Albany_CommTypes.hpp

```diff
@@ -13,17 +13,10 @@
 // Get Teuchos Comm type
 #include "Teuchos_DefaultComm.hpp"
 
-#ifdef ALBANY_MPI
-  using Albany_MPI_Comm = MPI_Comm;
+using Albany_MPI_Comm = MPI_Comm;
```

Comment:
This is only used in CommUtils, and given that there's no conditional compilation anymore, I would prune it, replacing it with `MPI_Comm` in the comm utils files.

Reply:
The reason is clarity: a typedef suggests the type might be complex and/or dependent on other stuff, so the user might spend extra time grepping for it, to figure out what's the catch, and why this is not just MPI_Comm.